integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/mail/EduMailSender.java

```diff
@@ -1,12 +1,18 @@
 package no.difi.meldingsutveksling.mail;
 
+import com.sun.mail.smtp.SMTPTransport;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1191)